### PR TITLE
Implemented sysops tasks: create, generate, exec, install

### PR DESCRIPTION
**Pull request by D1reCT**

### CODE REVIEW

File: bin/kaish [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+
+lib = File.expand_path('../../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+require 'kaigara/application'
```
Comment (mod) [inline bin/kaish]: require 'kaigara'

File: lib/kaigara.rb [hunk 2]
```diff
@@ -1,5 +1,9 @@
 require "kaigara/version"
```
Comment (mod) [inline lib/kaigara.rb]: remove this bundler bloc

File: lib/kaigara.rb [hunk 3]
```diff
@@ -1,5 +1,9 @@
...
```
Comment (mod) [inline lib/kaigara.rb]: require 'thor'

require 'kaigara/application'

File: lib/kaigara/application.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+require 'kaigara/sysops'
+
+module Kaigara
+  class Application < Baseops
```
Comment (mod) [inline lib/kaigara/application.rb]: Application is not a Baseops
no need to inherit for now

File: lib/kaigara/application.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
+require 'kaigara/sysops'
```
Comment (mod) [inline lib/kaigara/application.rb]: move to kaigara.rb for now

File: lib/kaigara/baseops.rb [hunk 6]
```diff
@@ -0,0 +1,8 @@
+module Kaigara
+  class Baseops < Thor
```
Comment (mod) [inline lib/kaigara/baseops.rb]: you can call this BaseController 
any task will inherit from BaseController

File: lib/kaigara.rb (hunk 3 above)
Comment (ichornous) [inline lib/kaigara.rb]: One doesn't need to require 'thor' if it's stated in Gemfile

File: spec/kaigara_spec.rb [hunk 7]
```diff
@@ -1,7 +1,39 @@
 require 'spec_helper'
```
Comment (mod) [inline spec/kaigara_spec.rb]: you inverted the files
specs/kaish_spec.rb for testing the kaish command line

File: spec/kaish_spec.rb [hunk 8]
```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+require 'kaigara/sysops'
+
```
Comment (mod) [inline spec/kaish_spec.rb]: this file should be named specs/kaigara_spec.rb

File: lib/kaigara/baseops.rb (hunk 6 above)
Comment (D1reCT) [inline lib/kaigara/baseops.rb]: But you told me to call it baseops
https://github.com/helios-technologies/kaigara/pull/1#discussion_r62297249

File: lib/kaigara/application.rb (hunk 4 above)
Comment (D1reCT) [inline lib/kaigara/application.rb]: But it uses thor. So should I inherit it from Thor?

File: lib/kaigara/baseops.rb (hunk 6 above)
Comment (mod) [inline lib/kaigara/baseops.rb]: true I see, I was thinking base common class to all tasks

**Pull request closed without merging**